packages/dashboard/src/vaadin-dashboard-section.js

```diff
@@ -166,6 +165,12 @@ class DashboardSection extends DashboardItemMixin(ElementMixin(ThemableMixin(Pol
         value: '',
       },
 
+      /* @private */
+      __rootHeadingLevel: {
+        type: Number,
+        value: 2,
```

Comment:
Default seems unnecessary here, given that we use a fallback anyway.

Reply:
Removed.